src/components/user/TargetForm.js

```diff
@@ -77,7 +79,7 @@ let TargetForm = ({ error, handleSubmit, submitting, intl, topics, eraseTarget }
                 classNamePrefix="react-select"
                 component={SelectBox}
                 label={intl.formatMessage(messages.topic)}
-                name="topic"
+                name="topic_id"
```

Comment:
why are you using snake_case? the apiService already transforms it from camelCase to snake_case before sending it to the backend.

Reply:
True! Forgot about that.